documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -13,11 +13,20 @@ When applied, the configuration creates a dedicated ingress and service for an e
 Clients connect to the bootstrap ingress, which routes them through the bootstrap service to connect to a broker. 
 Per-broker connections are then established using DNS names, which route traffic from the client to the broker through the broker-specific ingresses and services.
 
-To connect to a broker, you specify a hostname for the ingress bootstrap address, as well as the TLS certificate.
-Authentication is optional.
+IMPORTANT: The Ingress NGINX Controller for Kubernetes is not supported on OpenShift. 
+For Strimzi on OpenShift, use route listeners instead.
```

Comment:
I would leave this out. It is supported on OCP in the same way as on Kubernetes. The OpenShift routes are just easier as they are already there without the need to install anything new.

Reply:
removed